conda/recipes/librmm/meta.yaml

```diff
@@ -24,15 +25,15 @@ requirements:
 build:
   script_env:
     - PARALLEL_LEVEL
-    - CMAKE_GENERATOR=Ninja
+    - CMAKE_GENERATOR
```

Comment:
If we are removing the `CMAKE_GENERATOR=ninja` here where does it get specified for builds?

Reply:
It's not clear in the changes from this PR, but `export CMAKE_GENERATOR=Ninja` is specified in both of the [build_cpp.sh](https://github.com/ajschmidt8/rmm/blob/3734c3bc9ddc71c6c75da6c57c66ae101bb0bec5/ci/build_cpp.sh#L7) and [build_python.sh](https://github.com/ajschmidt8/rmm/blob/3734c3bc9ddc71c6c75da6c57c66ae101bb0bec5/ci/build_python.sh#L7) scripts that start the `conda build` process.

Placing it here might be confusing for `rmm`, which only has a single C++ and Python recipe. But this pattern works better for single-sourcing the `CMAKE_GENERATOR=Ninja` declaratation for other repositories (like cudf, below) that have multiple conda recipes in their build scripts.

- https://github.com/ajschmidt8/cudf/blob/919eab8c323e6276d5d551c68c8e19245af00ec3/ci/build_python.sh#L8-L43

Reply:
The user could still override it (say when building locally). That said, agree it is better to drop it. Other changes may be needed in the recipe anyways

We may want to revisit some of these other environment variables as well in another issue/PR